app/views/stories/_main_stories_feed.html.erb

```diff
@@ -1,11 +1,5 @@
 <% if @default_home_feed && user_signed_in? %>
   <% cache("fetched-home-articles-object", expires_in: 3.minutes) do %>
-    <% @new_stories = Article.published.
```

Comment:
These are taken out because they'll be coming via JSON from the endpoint right?

Reply:
Kind of: the same query is being done, just at `app/controllers/stories_controller.rb:114` now instead of in the view. Instead of two queries in the view, I'll be able to replace it with a single endpoint call at some point :)